README.md

```diff
@@ -76,6 +76,7 @@ puts decoded_token
 * HS512 - HMAC using SHA-512 hash algorithm
 
 ```ruby
+# The secret is required. A JWT::DecodeError will be raised if it isn't present.
```

Comment:
This is not totally true. You can try to decode with an empty secret. It just needs to be a string. `nil` does not work.

Reply:
```suggestion
# The secret must be a string. A JWT::DecodeError will be raised if it isn't provided.
```